### PR TITLE
Geminus-Invasion fixes and tweaks

### PR DESCRIPTION
**Pull request by caelaislinn**
Changes:
- Fixes Geminus UNSC jobs
- Reorders the simple mob spawn weightings for factions, because it affects capture node 
defenders. 
- Adds James spirit flyby sound effect on Crusade

:cl: CaelAislinn
bugfix: Undeletes the UNSC
/:cl:

### CODE REVIEW

File: code/modules/halo/factions/player_factions.dm [hunk 1]
```diff
@@ -9,12 +9,18 @@
 	ship_types = list(/obj/effect/overmap/ship/npc_ship/combat/covenant/medium_armed,/obj/effect/overmap/ship/npc_ship/combat/covenant/heavily_armed)
 	defender_mob_types = list(
```
Comment (X0-11) [inline code/modules/halo/factions/player_factions.dm:10]: add spartans to unsc defender mobs?

Comment (caelaislinn) [reply code/modules/halo/factions/player_factions.dm:10]: Yeah they already are

Comment (X0-11) [reply code/modules/halo/factions/player_factions.dm:10]: perfect

File: maps/faction_bases/ODP_Cassius/spawnpoint.dm [hunk 2]
```diff
@@ -13,13 +13,13 @@ GLOBAL_LIST_EMPTY(unsc_base_fallback_spawns)
 
 /datum/spawnpoint/unsc_base_fallback
 	display_name = "UNSC Base Fallback Spawns"
+	restrict_job_type = list(/datum/job/unsc)
```
Comment (X0-11) [inline maps/faction_bases/ODP_Cassius/spawnpoint.dm]: why is restrict_job_type defined twice

Comment (caelaislinn) [reply maps/faction_bases/ODP_Cassius/spawnpoint.dm]: fixed

**Review by X0-11 — APPROVED** [review]

**Pull request merged**